rust/datafusion/src/execution/dataframe_impl.rs

```diff
@@ -232,6 +238,23 @@ mod tests {
         Ok(())
     }
 
+    #[test]
+    fn registry() -> Result<()> {
+        // build query with a UDF using DataFrame API
```

Comment:
this is now useless, as built-in functions no longer require the registry. We should register one and use it.